kin-sdk/Environment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using kin_base;
+
+namespace kin_sdk
+{
+    public class Environment
+    {
+
+        public readonly string networkUrl;

```

Comment:
public properties are typical PascalCase in C#

```
NetworkUrl
NetworkPassphrase
```
also the below can be used to make a readonly property.
```
public string NetworkUrl {get;}
```